### ISSUE TITLE
Update the LuaJIT build with the new module handling

### ISSUE THREAD

**Issue by trink** [mozilla-services/lua_sandbox#66]

**Comment by jestan** [comment]
0.9.0 build with LuaJIT fails with following error messages.

```
[ 81%] Performing build step for 'lua_sandbox'
[  1%] Building C object src/CMakeFiles/luasandbox.dir/lsb.c.o
/heka/build/ep_base/Source/lua_sandbox/src/lsb.c: In function ‘lsb_init’:
/heka/build/ep_base/Source/lua_sandbox/src/lsb.c:347:20: error: ‘LUA_GCSETMEMLIMIT’ undeclared (first use in this function)
   lua_gc(lsb->lua, LUA_GCSETMEMLIMIT,
                    ^
/heka/build/ep_base/Source/lua_sandbox/src/lsb.c:347:20: note: each undeclared identifier is reported only once for each function it appears in
/development/heka/build/ep_base/Source/lua_sandbox/src/lsb.c: In function ‘lsb_usage’:
/development/heka/build/ep_base/Source/lua_sandbox/src/lsb.c:418:31: error: ‘LUA_GCMAXCOUNT’ undeclared (first use in this function)
       return lua_gc(lsb->lua, LUA_GCMAXCOUNT, 0) * 1024
                               ^
/heka/build/ep_base/Source/lua_sandbox/src/lsb.c:419:33: error: ‘LUA_GCMAXCOUNTB’ undeclared (first use in this function)
              + lua_gc(lsb->lua, LUA_GCMAXCOUNTB, 0);
                                 ^
make[5]: *** [src/CMakeFiles/luasandbox.dir/lsb.c.o] Error 1
make[4]: *** [src/CMakeFiles/luasandbox.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [ep_base/Stamp/lua_sandbox/lua_sandbox-build] Error 2
make[1]: *** [CMakeFiles/lua_sandbox.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by trink** [comment]
Yes this is expected, it will be corrected but is low priority at the moment.

**Comment by jestan** [comment]
@trink  I need to write a plugin using lua (mmap fileoutput plugin), but required dependencies are using luajit. I wish this issue will be fixed soon :)

**Comment by waqark3389** [comment]
Any idea when this will be added back in?

**Comment by trink** [comment]
This will not be in the 1.0 release
